libs/resource_partitioner/tests/unit/resource_partitioner_info.cpp

```diff
@@ -47,10 +47,10 @@ int main(int argc, char* argv[])
 {
     std::vector<std::string> cfg = {"hpx.os_threads=4"};
 
-    // set up the resource partitioner
-    hpx::resource::partitioner rp(argc, argv, std::move(cfg));
+    hpx::init_params init_args;
+    init_args.cfg = std::move(cfg);
 
     // now run the test
-    HPX_TEST_EQ(hpx::init(rp), 0);
+    HPX_TEST_EQ(hpx::init(1, init_args.dummy_argv, init_args), 0);
```

Comment:
```suggestion
    HPX_TEST_EQ(hpx::init(init_args), 0);
```
If we don't have this overload we should have it. It can be `hpx::init(init_params = {} /*or some reasonable default*/)` in which case it also covers `hpx::init()`.

Reply:
Yeah we have it I just forgot the parameter in there !